daemon/src/model/cfd.rs

```diff
@@ -254,9 +254,9 @@ pub struct Cfd {
 }
 
 impl Cfd {
-    pub fn new(cfd_offer: CfdOffer, quantity: Usd, state: CfdState, position: Position) -> Self {
+    pub fn new(cfd_offer: Order, quantity: Usd, state: CfdState, position: Position) -> Self {
```

Comment:
This variable still says `cfd_offer`.

Reply:
crap, there are a few that still say it ... - fixing

Reply:
done